classes/handlers/keywordsBehaviorHandler.ts

```
@@ -124,6 +145,45 @@ const scriptHandlers = {
+          return;
+        }
+
+        const preview = result.userMessage.replace(/\s+/g, ' ').trim();
+        const previewSnippet = preview.length > 120 ? `${preview.slice(0, 117)}…` : preview;
+        const amnesiaEmbed = new EmbedBuilder()
+          .setColor('#57F287')
+          .setTitle('Amnesia')
+          .setDescription(
+            `Forgot the last turn from **${displayName}** session **#${result.sessionId}**`
+            + ` (${result.deletedCount} row${result.deletedCount === 1 ? '' : 's'}).`
+            + (previewSnippet ? `\n\n> ${previewSnippet}` : ''),
```

ESLint

[error] Unexpected string concatenation.

(prefer-template)